build.psm1

```diff
@@ -1151,14 +1164,42 @@ function Start-PSPester {
         }
     }
 
+    $PSFlags = @("-noprofile")
+    if (-not [string]::IsNullOrEmpty($ExperimentalFeatureName)) {
+        $configFile = [System.IO.Path]::GetTempFileName()
+        $configFile = [System.IO.Path]::ChangeExtension($configFile, ".json")
+
+        if ($Environment.IsWindows) {
+            $content = @"
+{
+    "Microsoft.PowerShell:ExecutionPolicy":"RemoteSigned",
```

Comment:
It would be good to have a comment here about why there is 'RemoteSigned' difference between Windows/Non-Windows code paths.

Reply:
Will add a comment.